CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 4.0.0
+* Added support for setDeviceAttributes 
```

Comment:
can you add a note like 
**BREAKING CHANGE** - multi method signature has changed.